04.class/operateInterface.js

```diff
@@ -0,0 +1,97 @@
+import readline from "readline";
+import enquirer from "enquirer";
+
+const { Select } = enquirer;
+export default class OperateInterface {
+  constructor(memoOperation) {
+    this.memoOperation = memoOperation;
+  }
+
+  async listMemos() {
+    try {
+      const rows = await this.memoOperation.fetchMemos();
+      rows.forEach((row) => {
+        console.log(row.content.split("\n")[0]);
+      });
+    } catch (err) {
+      console.error(err);
+    }
+  }
+
+  async readMemos() {
+    try {
+      const result = await this.#selectMemosFromList("see");
```

Comment:
```suggestion
      const content = await this.#selectMemosFromList("see");
```

結果として返ってくるものはmemoのcontentなのでは？であればcontentと書いた方が具体的でわかりやすいと思います。（具体 vs 抽象のバランスが大事）
https://bootcamp.fjord.jp/articles/78

Reply:
抽象的すぎるものは理解できている気がしていますが、過度な具体性については今後発生させてしまいそうだと思いました。
今回も`memoContent`の方が良いのかな？と一瞬考えたので、練習を重ねて身につけられたらと思います。